config/localization/en/app.json

```diff
@@ -950,6 +950,7 @@
       "EXPECTED_AMOUNT_RECEIVED": "Expected Amount Received",
       "FAST": "Fast",
       "FAST_WITHDRAW_CONFIRMATION": "A fee of {FEE} will be collected for this fast withdraw. Your funds will be sent immediately and will appear in your wallet once the withdraw transaction has confirmed.",
+      "LOWEST_FEE_WITHDRAWALS": "USDC withdrawals to {LOWEST_FEE_TOKENS_TOOLTIP} have the lowest fees. Other withdrawals may have additional third-party fees",
```

Comment:
considering changing `third-party` to `bridge` since thats the nomenclature we've been using in the modals so far

Reply:
nit: add a period at the end